<launch>
  <node name="publisher" pkg="asctec_autopilot" type="publisher" respawn="false" output="screen">
    <param name="serial_port" type="string" value="/dev/ttyUSB0"/>
    <param name="serial_speed" type="int" value="57600"/>
  </node>
</launch>
